installer/lib/phx_new/generator.ex

```diff
@@ -349,10 +349,12 @@ defmodule Phx.New.Generator do
           environment variable DATABASE_URL is missing.
           For example: ecto://USER:PASS@HOST/DATABASE
           \"""
+
+      ipv6? = System.get_env("IPV6") == "true"
```

Comment:
Is `IPV6` the correct environment variable here? Since this is specific to the db socket, wouldn't something like `DATABASE_IPV6` be more appropriate?